docs/ros/ros/installation/remote_pc.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Remote PC setup
+sidebar_label: Remote PC
+sidebar_position: 2
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
```

Comment:
Should we add one sentence here to indicate when/why a remote computer would be used?